From 72b339bbc889f95a8c4acc2577885294913d9116 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 8 Aug 2022 08:26:30 +0300
Subject: [PATCH 073/102] s32cc: Remove platform SerDes initialization

This isn't needed anymore considering that all SerDes phys
are obtained an initialized through generic phy API.

Issue: ALB-8357
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/mach-s32/s32-cc/soc.c | 18 ------------------
 1 file changed, 18 deletions(-)

diff --git a/arch/arm/mach-s32/s32-cc/soc.c b/arch/arm/mach-s32/s32-cc/soc.c
index 62479f001c..59990f5389 100644
--- a/arch/arm/mach-s32/s32-cc/soc.c
+++ b/arch/arm/mach-s32/s32-cc/soc.c
@@ -116,23 +116,6 @@ int arch_cpu_init(void)
 	return 0;
 }
 
-static void serdes_init(void)
-{
-	struct udevice *bus = NULL;
-
-	/*
-	 * Enumerate all known UCLASS_PHY devices. This will
-	 * also probe them, so the SerDes devices will be enumerated too.
-	 * TODO: Enumerate first the EPs, so that loopback between
-	 * the two PCIe interfaces will also work if PCIe1 is EP.
-	 */
-	for (uclass_first_device(UCLASS_PHY, &bus);
-	     bus;
-	     uclass_next_device(&bus)) {
-		;
-	}
-}
-
 __weak void show_pcie_devices(void)
 {
 }
@@ -151,7 +134,6 @@ int initr_pci(void)
 		}
 	}
 
-	serdes_init();
 	/*
 	 * Enumerate all known PCIe controller devices. Enumeration has
 	 * the side-effect of probing them, so PCIe devices will be
-- 
2.17.1

